conf/messages.en

```diff
@@ -98,7 +98,7 @@ primaryContactDetails.fullNamePage.lastName.error.empty = Enter a last name
 primaryContactDetails.fullNamePage.lastName.error.length = Last name cannot be more than 20 characters long
 primaryContactDetails.fullNamePage.lastName.error.specialCharacters = Enter a last name in the correct format
 
-primaryContactDetails.jobTitlePage.title = What is the primary contact's job title?
```

Comment:
Are we going to use start using curly single quotes?

Reply:
yeap, apparently if you use straight quote it's not displayed on FE